docs/reference/objects/product/payment_plan.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: PaymentPlan
+parent: Product
+has_children: false
+---
+
+A `payment_plan` has the following attributes:
+
+# payment_plan.number_of_instalments
```

Comment:
[Suggestion] I think we've alphabetised these elsewhere so might be worth following that convention.